Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -480,12 +480,16 @@ static Color activeButtonTextColor()
     auto color = [this, cssValueID, options, useDarkAppearance]() -> Color {
         LocalDefaultSystemAppearance localAppearance(useDarkAppearance);
 
-        auto selectCocoaColor = [cssValueID] () -> SEL {
+        auto selectCocoaColor = [cssValueID, &localAppearance] () -> SEL {
```

Comment:
You can just capture `useDarkAppearance`?

Reply:
I wanted to be consistent with the code below, which always uses `localAppearance`